### DISCUSSION TITLE
1.64 fails to allocate memory or throws "exception: access violation reading 0x0000001056C88000" on generation

### DISCUSSION THREAD

**Discussion by KKITAANIK** [YellowRoseCx/koboldcpp-rocm#45] [General]
Pretty much exactly what it says in the title. Presets that worked in 1.61.2 now simply can't be launched. Depending on my settings they'll either fail at `ggml_cuda_host_malloc`, and when it's done setting up it crashes when trying to run koboldcpp because of the unhandled exception, or it'll run fine and then throw an access violation after processing context and trying to generate the first token.

**Comment by strider16** [comment]
Experiencing the same with 1.65.
